no-littering.el

```diff
@@ -234,6 +234,7 @@ This variable has to be set before `no-littering' is loaded.")
        `(make-directory ,(var "elfeed/") t))
     (setq elfeed-db-directory              (var "elfeed/db/"))
     (setq elfeed-enclosure-default-dir     (var "elfeed/enclosures/"))
+    (setq rmh-elfeed-org-files             (list (var "elfeed/elfeed.org")))
```

Comment:
(21) What if I mention its hash when commenting on that commit? That would be dd162629c6bc3d06bd8dca9cf800a75d58475c08 at this time. Using "Add single comment".